Dev/VideoChatClient/Common/Common.hpp

```diff
@@ -4,3 +4,22 @@
 #include <vector>
 
 using Buffer = std::vector<uint8_t>;
+
+
+enum class PacketDataTypes : uint16_t
+{
+    UninitDataType = 0,
+	TextDataType,
+	ImageDataType,
+	AudioDataType,
+};
+
+struct PacketHeaders            // 80 bytes
+{
+    char RT_Magic[2];           // 2 bytes
+    uint32_t meeting_id;        // 4 bytes
+    uint32_t user_unique_id;    // 4 bytes
+    char name[64];              // 64 bytes
```

Comment:
Use std::array of this size, magic number here too

Reply:
We can't do that, since we are writing to it directly so we can't use a class